### PR TITLE
Front end

### PR DESCRIPTION
**Pull request by cagric0**
Guys, you need to review it as soon as possible.

### CODE REVIEW

**Comment by burakcuhadar** [comment]
Book search and astronomy image functionalities do not work on my machine. When I search for a book I do not get any results and an astronomy image is not displayed at the bottom of the page.

**Comment by berkayalkan** [comment]
@burakcuhadar Did you update the .env file?

**Comment by burakcuhadar** [comment]
> @burakcuhadar Did you update the .env file?

No, I forgot to do that :D

File: practice-app/frontend/index.html [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+                </div>
+            </div>
+
+        </div>
+        <div class="col-xl col-lg-auto col-md-auto col-sm-auto bg-light">
+            <div class="container p-5 border">
+                <h2 id="news_header">NEWS</h2>
+                <div id="news_cont1" class="container p-5 border">
+                    <h5 id="news_title1"></h5>
+                    <p id="news_desc1">ds</p>
```
Comment (zeritte) [inline practice-app/frontend/index.html:76]: this "ds" should be removed

File: practice-app/frontend/index.html [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+                    <a id="news_link1" class="btn btn-primary" href="" role="button">Devamı</a>
```
Comment (zeritte) [inline practice-app/frontend/index.html:77]: "Devamı" should be "continue reading"

File: practice-app/frontend/index.html (hunk 1 above)
Comment (zeritte) [reply practice-app/frontend/index.html]: fixed

File: practice-app/frontend/index.html (hunk 2 above)
Comment (zeritte) [reply practice-app/frontend/index.html]: fixed

**Pull request merged**